code/datums/spells/ethereal_jaunt.dm

```diff
@@ -95,6 +95,7 @@
 	var/canmove = 1
 	density = 0
 	anchored = 1
+	invisibility = INVISIBILITY_OBSERVER

```

Comment:
wait what
Uh.

This means that only observers will be able to see the jaunt effect dummy. I'm pretty sure that's not supposed to be how it works.